cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -322,6 +322,33 @@ def _cleanup():
         pass
 
 
+@pytest.fixture
+def setup_user_for_v2v_migration(appliance, new_credential):
```

Comment:
this is a good fixture :+1: 
@sshveta and @nachandr what you think to put this as a global fixture 
https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/v2v_fixtures.py

Reply:
@nachandr : Please check if there is already a fixture available like this . If not we can make it a global fixture .
Sure it can be used in all rbac tests